Enable text_overrun for project export errors

TrueJole opened the pull request:
The errors shown at the bottom of the project export window should now have text_overrun_behavior.

- auto_wrap caused the window to change its height in an undesired manner
- should fix #103275 
- this is my first pull request, any advise is appreciated

TrueJole committed:
Update project_export.cpp: Autowrap on error labels

TrueJole committed:
Reverted change

bruvzg commented:
If I remember correctly, enabling autowrap for these labels was causing the export window to have excessive height. I'm not sure if the cause of this issue was fixed, but it's something that should be tested on all platforms.

TrueJole commented:
> If I remember correctly, enabling autowrap for these labels was causing the export window to have excessive height. I'm not sure if the cause of this issue was fixed, but it's something that should be tested on all platforms.

Yes, that is still the case for both Linux and Windows. I tried setting the text_overrun_behavior to  OVERRUN_TRIM_WORD_ELLIPSIS. This does not cause the window to change it's height and it is better than causing a very wide window.

TrueJole committed:
Merge branch 'godotengine:master' into master

TrueJole committed:
Update project_export.cpp: text_overrun for errors

Changed the auto_wrap that caused the window size to change to a text_overrun (OVERRUN_TRIM_WORD_ELLIPSIS)

TrueJole committed:
Merge branch 'godotengine:master' into master

TrueJole commented:
I am closing this PR to reopen it on another branch (on my side, so that its not on main) with only one commit. Git is a learning process I guess.

Calinou commented:
- Superseded by https://github.com/godotengine/godot/pull/103598.
